supplementary_style_guide/glossary_terms_conventions/general_conventions/b.adoc

```diff
@@ -78,6 +78,15 @@
 
 *See also*: xref:distinguished-name[distinguished name]
 
+[discrete]
+[[baseos-repository]]
+==== image:images/yes.png[yes] BaseOS repository (noun)
+*Description*: In Red Hat Enterprise Linux, the  _BaseOS repository_ contains the core set of the underlying OS functionality that provides the foundation for all installations.
```

Comment:
Agree with Rolfe that there's something about "core set of ...  functionality" that doesn't quite jibe.

Reply:
As per the discussion with SME, leaving the original version.